openslides_backend/permissions/management_levels.py

```diff
@@ -1,15 +1,25 @@
 from enum import Enum
+from typing import Optional
 
 from .base_classes import VerbosePermission
 
 
 class CompareRightLevel(str, VerbosePermission, Enum):
-    def __new__(cls, value: str, weight: int):  # type: ignore
+    def __new__(
+        cls, value: Optional[str], weight: Optional[int] = None
+    ) -> "CompareRightLevel":
         obj = str.__new__(cls, value)  # type: ignore
         obj._value_ = value
         obj.weight = weight
         return obj
 
+    @classmethod
+    def _missing_(cls, value: object) -> "CompareRightLevel":
+        """
+        Always return the first enum item if no matching one was found. -> NO_RIGHT must always be listed first.
+        """
+        return next(iter(cls))
```

Comment:
Proposal: Define NO_RIGHT in this parent class with default value ("no_right") and use it here instead of implicit catch of first enum entry (to prevent security issues if a further developer does not keep this in mind.

Reply:
Unfortunately, python does not allow subclassing of enums which define members: https://docs.python.org/3/library/enum.html#restricted-enum-subclassing